luigi_pipeline/requirements.txt

```diff
@@ -0,0 +1,3 @@
+elasticsearch==6.2.0
```

Comment:
elasticsearch>=6.2.0  ?

Reply:
Just copied and pasted from https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/master/requirements.txt so not sure if we needed to pin it.

Reply:
There's also several scripts that call `os.system("pip install elasticsearch")` without _any_ version requirement.

Reply:
How about I just merge this requirements file with the other one and remove it here so we don't have redundancy?

Reply:
That sounds good. Though there may also be cause for distinguishing what packages are needed locally to submit jobs vs what packages are needed on the Spark cluster.

Reply:
@knguyen142  merging this into the top-level requirements file is a good idea.
@nawatts that's a good point. The os.system("pip install elasticsearch") should only be present in scripts that run on Spark - not sure if there's a better way to install dependencies there. I don't have a preference for `>=` vs. `==` for lib. versions. Do you guys?

Reply:
A cluster start script could parse a `requirements.txt` file and call `cluster start --packages=...`